### DISCUSSION TITLE
Use BarToggler without BarBrand

### DISCUSSION THREAD

**Discussion by wilberolive** [Megabit/Blazorise#2382] [Q&A]
If you look at the side bar toggler on the [demo site](https://bootstrapdemo.blazorise.com/) you will notice it is next to a bar brand. If you click the toggler to collapse the side bar, the toggler will stay at the top and the brand logo goes below it. You'll also notice the menu items below move up and down as the side bar is opened/closed.

I'm trying to figure out how to do this without the bar brand at all. I just want both the toggler and the menu items to stay where they are when the side bar is opened/closed. I don't want to have a brand and logo in the side bar. The problem is, if I remove the BarBrand, then when the side bar is opened, the menu items move up and the toggler moves to the right next to them. They overlap and it looks ugly.

![toggler](https://user-images.githubusercontent.com/5822215/118753516-888eab80-b8a8-11eb-957e-24afa19001db.png)

**Comment by stsrki** [comment]
Hi. A similar problem is already reported and is scheduled for the next patch. https://github.com/Megabit/Blazorise/issues/2330

Until I manage to fix it I think it's best to workaround it with CSS, if possible.
